### ISSUE TITLE
Failure in test case of test_format_instances_as_rows()

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#782]
From PR #778, [Travis build 4447](https://travis-ci.org/github/pywbem/pywbemtools/builds/732863870):

```
_ test_format_instances_as_rows[Verify 2 instances where second one has path-kwargs5-None-None-True] _
testcase = testcase_tuple(desc='Verify 2 instances where second one has path', kwargs={'args': (), 'kwargs': {'insts': [CIMInstan...'"VP1a"', '"VP2a"'], ['"K1b"', '"K2b"', '"VP1b"', '"VP2b"']]}, exp_exc_types=None, exp_warn_types=None, condition=True)
args = ()
kwargs = {'insts': [CIMInstance(classname='CIM_Foo', path=None, properties=NocaseDict({'P2': CIMProperty(name='P2', value='VP2a...=False, array_size=None, class_origin=None, propagated=None, qualifiers=NocaseDict({}))}), qualifiers=NocaseDict({}))]}

exp_rtn = [['', '', '"VP1a"', '"VP2a"'], ['"K1b"', '"K2b"', '"VP1b"', '"VP2b"']]
    @pytest.mark.parametrize(
        "desc, kwargs, exp_exc_types, exp_warn_types, condition",
        TESTCASES_FORMAT_INSTANCES_AS_ROWS)
    @simplified_test_function
    def test_format_instances_as_rows(testcase, args, kwargs, exp_rtn):
        """
        Test the output of the common _format_instances_as_rows() function
        """
    
        # The code to be tested
        act_rtn = _format_instances_as_rows(*args, **kwargs)
    
        # Ensure that exceptions raised in the remainder of this function
        # are not mistaken as expected exceptions
        assert testcase.exp_exc_types is None
        # result is list of lists.  we want to test each item in inner list
    
        assert len(act_rtn) == len(exp_rtn), \
            "Unexpected number of lines in test desc: {}:\n" \
            "Expected line cnt={}:\n" \
            "{}\n\n" \
            "Actual line cnt={}:\n" \
            "{}\n". \
            format(testcase.desc, len(act_rtn), '\n'.join(act_rtn),
                   len(exp_rtn), '\n'.join(exp_rtn))
    
>       assert exp_rtn == act_rtn, \
            "Unequal values for test desc: {}:\n" \
            "Expected = {}:\n" \
            "Actual   = {}:\n". \
            format(testcase.desc, exp_rtn, act_rtn)
E       AssertionError: Unequal values for test desc: Verify 2 instances where second one has path:
E         Expected = [['', '', '"VP1a"', '"VP2a"'], ['"K1b"', '"K2b"', '"VP1b"', '"VP2b"']]:
E         Actual   = [['"VP1a"', '"VP2a"', '', ''], ['"VP1b"', '"VP2b"', '"K1b"', '"K2b"']]:
E         
E       assert [['', '', '"V..."', '"VP2b"']] == [['"VP1a"', '...b"', '"K2b"']]
E         At index 0 diff: ['', '', '"VP1a"', '"VP2a"'] != ['"VP1a"', '"VP2a"', '', '']
E         Use -v to get the full diff
tests/unit/test_display_cimobjects.py:444: AssertionError
```

It seems the culprit is `sorted_prop_names()` which returns an incorrectly ordered property list for this test case, because it does not properly recognize key properties (that should go first) and thus sorts them all by "lower cased" property name.

**Comment by andy-maier** [comment]
Trying to investigate why the bug did not surface earlier:

* success in master branch as of yesterday: [Travis build 4444](https://travis-ci.org/github/pywbem/pywbemtools/builds/732769594).
* same failure in PR #783 that is same as master as of yesterday: [Travis build 4460](https://travis-ci.org/github/pywbem/pywbemtools/builds/732910282)

**Comment by andy-maier** [comment]
The fix in PR #785 however is understood, so I suggest we go forward with it.
